src/main/java/core/basesyntax/service/operation/ReturnOperationHandler.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.StorageService;
+
+public class ReturnOperationHandler implements OperationHandler {
+    private final StorageService storageService;
+
+    public ReturnOperationHandler(StorageService storageService) {
+        this.storageService = storageService;
+    }
+
+    @Override
+    public void handle(FruitTransaction transaction, StorageService storageService) {
```

Comment:
The `handle` method should not have `StorageService` as a parameter since `ReturnOperationHandler` has a `storageService` field. This is likely a mistake and could lead to using the wrong `StorageService` instance. Remove the parameter and use the `storageService` field instead.